pandas/core/generic.py

```diff
@@ -6671,9 +6671,11 @@ def replace(
             0.
         inplace : bool, default False
             Update the data in place if possible.
-        limit_direction : {'forward', 'backward', 'both'}, default 'forward'
+        limit_direction : {'forward', 'backward', 'both'}, default is 'forward'
```

Comment:
the default is now 'backward' for method 'backfill' and 'bfill'?

Reply:
@cchwala can you address

Reply:
already change after merging master